Managers/LuaMan.h

```diff
@@ -162,6 +162,22 @@ namespace RTE {
 #pragma endregion
 
 #pragma region File I/O Handling
+		/// <summary>
+		/// Returns an iterator over all the directories in relativeDirectory, which is relative to the working directory.
```

Comment:
Neither of these return iterators, and m_Paths doesn't store iterators either, just a vector of strings.
They're iterators in Lua, which is irrelevant and incorrect here.